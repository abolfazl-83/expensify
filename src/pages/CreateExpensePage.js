import React from 'react';
import CreateExpense from '../components/CreateExpense';

export const CreateExpensePage = (props) => {
    return (
        <React.Fragment>
            <CreateExpense {...props}/>
        </React.Fragment>
    );
};

export default CreateExpensePage;