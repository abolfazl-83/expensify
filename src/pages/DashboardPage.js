import React from 'react';
import ExpensesList from '../components/ExpensesList';
import ExpensesSummery from '../components/ExpensesSummery';
import ExpensesFilters from '../components/ExpensesFilters';

export const DashboardPage = () => {
    return (
        <div>
            <ExpensesSummery />
            <ExpensesFilters />
            <ExpensesList />
        </div>
    );
};

export default DashboardPage;