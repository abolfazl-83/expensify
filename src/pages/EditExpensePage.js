import React from 'react';
import EditExpense from '../components/EditExpense';

export const EditExpensePage = (props) => {
    return (
        <React.Fragment>
            <EditExpense {...props}/>
        </React.Fragment>
    );
};

export default EditExpensePage;