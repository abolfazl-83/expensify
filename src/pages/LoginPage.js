import React from 'react';
import Login from '../components/Login';

export const LoginPage = () => {
    return (
        <React.Fragment>
            <Login />
        </React.Fragment>
    );
};

export default LoginPage;