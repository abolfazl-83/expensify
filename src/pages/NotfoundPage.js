import React from 'react';
import Notfound from '../components/Notfound';

export const NotfoundPage = (props) => {
    return (
        <React.Fragment>
            <Notfound {...props} />
        </React.Fragment>
    );
};

export default NotfoundPage;