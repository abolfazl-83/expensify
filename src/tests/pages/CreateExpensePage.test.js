import React from 'react';
import {shallow} from 'enzyme';
import {CreateExpensePage} from '../../pages/CreateExpensePage';

test('should render CreateExpensePage correctly' , () => {
    let props = {
        history : {
            push : jest.fn()
        }
    }
    const wrapper = shallow(<CreateExpensePage {...props} />);
    expect(wrapper).toMatchSnapshot();
});