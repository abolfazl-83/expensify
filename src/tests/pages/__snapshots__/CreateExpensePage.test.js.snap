// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CreateExpensePage correctly 1`] = `
<Fragment>
  <Connect(CreateExpense)
    history={
      Object {
        "push": [MockFunction],
      }
    }
  />
</Fragment>
`;
